ingestion_server/ingestion_server/indices/mappings/common.json

```diff
@@ -0,0 +1,96 @@
+{
+  "authority_boost": {
+    "type": "rank_feature"
+  },
+  "authority_penalty": {
+    "positive_score_impact": false,
+    "type": "rank_feature"
+  },
+  "category": {
+    "type": "keyword"
+  },
+  "created_on": {
+    "type": "date"
+  },
+  "creator": {
+    "fields": {
+      "keyword": {
+        "ignore_above": 256,
+        "type": "keyword"
+      }
+    },
+    "type": "text"
+  },
+  "description": {
+    "analyzer": "custom_english",
+    "fields": {
+      "keyword": {
+        "ignore_above": 256,
+        "type": "keyword"
+      }
+    },
+    "similarity": "boolean",
+    "type": "text"
+  },
+  "extension": {
+    "type": "keyword"
+  },
+  "id": {
+    "type": "long"
+  },
+  "identifier": {
+    "type": "keyword"
+  },
+  "license": {
+    "type": "keyword"
+  },
+  "license_version": {
```

Comment:
I don't think we will ever search for licenses by version. We could remove this mapping.

Reply:
This is good, fix pushed in 134a743.